src/lib.rs

```diff
@@ -250,7 +258,7 @@ impl Surreal {
                 .range((range.beg, range.end)),
             Err(_) => self.db.update(Resource::from(resource)),
         };
-        let data: Value = from_value(data)?;
+		let data = json(&data.to_string()).unwrap();
```

Comment:
I know it's unlikely to run into an error here but it's possible. Kindly propagate it please.